docs/dev/developer.rst

```diff
@@ -64,15 +127,14 @@ Online tests will be run automatically. They will check for:
 Help
 ----
 
-If you encounter any problem, please `report an Issue <https://github.com/radis/radis/issues?utf8=%E2%9C%93&q=is%3Aissue>`_ on GitHub.
+If you encounter any problem:
 
-You can also ask for advice on the `Q&A forum <https://groups.google.com/forum/#!forum/radis-radiation>`__
-or the community chat:
-
-.. image:: https://badges.gitter.im/Join%20Chat.svg
-    :target: https://gitter.im/radis-radiation/community
-    :alt: Gitter
+1. Check if it's a known issue in our `GitHub Issues <https://github.com/radis/radis/issues>`__
+2. If not, please `open a new issue <https://github.com/radis/radis/issues/new/choose>`__ with:
+   - A quick summary
+   - Steps to reproduce
+   - Expected vs actual behavior
+   - Any relevant notes or attempted solutions
 
-.. image:: https://img.shields.io/badge/slack-join-green.svg?logo=slack
-    :target: https://radis.github.io/slack-invite/
-    :alt: Slack
+You can also ask for advice on the `Q&A forum <https://groups.google.com/forum/#!forum/radis-radiation>`__
+or join our community chat.
```

Comment:
```suggestion
```
@erwanp let's remove this group that is redondant with GitHub (for issues) and Slack

Reply:
Insert Slack and Gitter links here